change avatars to dynamic

Arielpopcorn opened the pull request.

Commit:
change avatars to dynamic

Commit:
change error and put all of them into class

delfipolito commented on src/components/Home/Contributors.js:
```diff
@@ -1,50 +1,38 @@
+  render() {
+    return (
+        <Container href="https://github.com/lorikeetui/lorikeet/graphs/contributors"
+        target="_blank">>
```
Can we remove the extra > ?

delfipolito commented on src/components/Home/Contributors.js:
```diff
@@ -1,50 +1,38 @@
+    return (
+        <Container href="https://github.com/lorikeetui/lorikeet/graphs/contributors"
+        target="_blank">>
+          {this.state.items.map(item => <Image src={item.avatar_url} />)}
```
In React, each child in an array or iterator should have a unique "key" prop, so we need to add one here in Image component. Maybe it could be key={item.id}

delfipolito reviewed (commented):
Hey Ariel! It looks awesome!! 🔥🔥🔥
I added two tiny comments, but otherwise it looks great!
Thanks for the hard work 💪🏽

Commit:
Remove <AragonApp />

Commit:
add key in Inmage component

Arielpopcorn commented:
Thank you Delfi:)
I changed them☺️
And I changed and pushed this as well!

Remove <AragonApp /> · Issue #8 · lorikeetui/website - GitHub
This component won’t be in Lorikeet. It can be replaced by BaseStyles and PublicUrl.Provider, see the change on the Aragon website: https://github.com/aragon/website/pull/72/files



Sent from my iPhone

> On Nov 14, 2018, at 20:01, delfipolito <notifications@github.com> wrote:
> 
> @delfipolito commented on this pull request.
> 
> Hey Ariel! It looks awesome!! 🔥🔥🔥
> I added two tiny comments, but otherwise it looks great!
> Thanks for the hard work 💪🏽
> 
> In src/components/Home/Contributors.js:
> 
> > +    fetch('https://api.github.com/repos/aragon/aragon-ui/contributors')
> +    .then(res => res.json())
> +    .then(json => {
> +      console.log(json)
> +      this.setState({
> +        items: json,
> +        loading: false
> +      })
> +    })
> +    .catch(err => console.log(err))
> +  };
> +
> +  render() {
> +    return (
> +        <Container href="https://github.com/lorikeetui/lorikeet/graphs/contributors"
> +        target="_blank">>
> Can we remove the extra > ?
> 
> In src/components/Home/Contributors.js:
> 
> > +    .then(res => res.json())
> +    .then(json => {
> +      console.log(json)
> +      this.setState({
> +        items: json,
> +        loading: false
> +      })
> +    })
> +    .catch(err => console.log(err))
> +  };
> +
> +  render() {
> +    return (
> +        <Container href="https://github.com/lorikeetui/lorikeet/graphs/contributors"
> +        target="_blank">>
> +          {this.state.items.map(item => <Image src={item.avatar_url} />)}
> In React, each child in an array or iterator should have a unique "key" prop, so we need to add one here in Image component. Maybe it could be key={item.id}
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

delfipolito reviewed (approved):
Nice!! Thanks @Arielpopcorn 🤩
